jupyterlab_bigquery/jupyterlab_bigquery/handlers.py

```diff
@@ -114,10 +126,57 @@ def get_table_details(client, table_id):
           'link': table.self_link,
           'num_rows': table.num_rows,
           'num_bytes': table.num_bytes,
-          'schema': get_schema(table.schema)
+          'schema': format_detail_schema(table.schema)
       }
   }
 
+def format_preview_fields(schema):
```

Comment:
general question: is it better to format things like dates, bytes, etc here in the backend, or is it better to pass them to the frontend as a json string and then parse and format that in the frontend?